src/publicodes-state/providers/simulationProvider/useSetComputedResults.ts

```diff
@@ -43,10 +46,25 @@ export function useSetComputedResults({
         acc[metric] = categories.reduce(
           (acc, category) => {
             acc.categories[category] = getNumericValue(category, metric)
+
+            const subcategories = getSubcategories(category)
+
+            if (!subcategories) return acc
+
+            acc.subcategories[category] = subcategories.reduce(
+              (subAcc, subcategory) => {
+                subAcc[subcategory] = getNumericValue(subcategory, metric)
+
+                return subAcc
+              },
+              {} as { [key in DottedName]: number }
```

Comment:
You are using `Record<DottedName, number>` everywhere else. Why change ?

Reply:
No idea, might have been a Copilot shenanigan